tests/e2e/api/transaction_submission.test.ts

```diff
@@ -26,7 +27,7 @@ describe("transaction submission", () => {
       const aptos = new Aptos(config);
       const alice = Account.fromPrivateKey({
         privateKey: new Ed25519PrivateKey({
-          hexInput: "0x5aba8dab1c523be32bd4dafe2cc612f7f8050ce42a3322b60216ef67dc97768c",
+          hexInput: ALICE_PRIVATE_KEY,
```

Comment:
these tests are going to change. We now test it against devnet and eventually will change it to run against localnet. We had these fix accounts because we couldn't use the fund query so we fund it using the cli and "created" it on devnet. 
We will probably not use any fix private keys to create an account but create and fund it on run time